src/converter-uis1.js

```diff
@@ -1,11 +1,12 @@
+/*eslint complexity:0 */
```

Comment:
I'd much prefer to see `/* eslint-disable complexity */` before the complex function and `/* eslint-enabled complexity */` after it.